docker/Dockerfile.archlinux

```diff
@@ -0,0 +1,17 @@
+ARG  from
+FROM ${from}
+
+ENV  CFLAGS=-w CXXFLAGS=-w
+
+RUN pacman -Sy --noconfirm \
+    diffutils \
+    fontconfig \
+    freetype2 \
+    gcc \
```

Comment:
I think this installs gcc 10. See [Dockerfile.focal](https://github.com/wkhtmltopdf/packaging/blob/0.12.6-2/docker/Dockerfile.focal) -- you'll have to install gcc 7 and symlink it correctly.

Reply:
thx, I will trying